orchagent/pfcactionhandler.cpp

```diff
@@ -404,16 +406,18 @@ PfcWdLossyHandler::~PfcWdLossyHandler(void)
 {
     SWSS_LOG_ENTER();
 
-    uint8_t pfcMask = 0;
+    uint8_t pfcTxMask = 0;
+    uint8_t pfcRxMask = 0;
 
-    if (!gPortsOrch->getPortPfc(getPort(), &pfcMask))
+    if (!gPortsOrch->getPortPfc(getPort(), &pfcTxMask, &pfcRxMask))
     {
         SWSS_LOG_ERROR("Failed to get PFC mask on port 0x%" PRIx64, getPort());
     }
 
-    pfcMask = static_cast<uint8_t>(pfcMask | (1 << getQueueId()));
+    pfcTxMask = static_cast<uint8_t>(pfcTxMask | (1 << getQueueId()));
```

Comment:
We shouldn't be modifying Tx Mask since it contains only priorities 3 and 4. We will setting tx mask to incorrect values on pfcwd storm restore if pfcwd was triggered on a lossy priority

Reply:
Agree, we don' need it. Fixed to modify only Rx mask.

Reply:
Thinking about this more, we need to set the txmask but need to do it only for queues set in 'pfc_enable' field

Reply:
Ok, I will change it back and verify